sdk/storage/azblob/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 * Updated to latest `azcore`.  Public surface area is unchanged.
 
 ### Bugs Fixed
+* Fixed Issue #16816 : ContainerClient.GetSASToken doesn't allow list permission.
```

Comment:
Two nits here: 
1. If you want to include the issue number, we should include a link in the format `[issue num] (link/to/issue)`
2. The code portions (ContainerClient.GetSASToken, etc.) should be enclosed in backticks (`)

Reply:
1. GitHub automatically links issue if given in the above format.

2. Yes, I'll add back-ticks to enclose code. Thanks for pointing this out.